### PR TITLE
[PERFSCALE-2497]: Synchronize prow AWS instance types

### PR DESCRIPTION
**Pull request by krishvoor**
This PR will bring consistency to our self-managed AWS jobs
by updating corresponding instance types.

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi @jtaleric: I have updated the Control Plane sizes for x86 one - can you ptal

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml [hunk 1]
```diff
@@ -29,34 +29,39 @@ tests:
...
-      SIZE_VARIANT: large
+      CONTROL_PLANE_INSTANCE_TYPE: m5.4xlarge
     post:
     - chain: ipi-aws-post
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-control-plane
-  timeout: 5h0m0s
+  timeout: 6h0m0s
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml:39]: Shall we do the same change for 4.15 release? 4.13 and 4.14 are all 6h now, 5.15 is still 5h.

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml:39]: Ack, will update the timeout in 4.15 as well!

**Comment by jtaleric** [comment]
@krishvoor I had a chat with AndrewC about this. Should we rip off the m6 bandaid and move to m6 vs m5? Much of our previous results were m6 masters...

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml [hunk 2]
```diff
@@ -29,9 +29,9 @@ tests:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "252"
       COMPUTE_NODE_TYPE: m5.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.8xlarge
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml:32]: Not updating these to m6i family, as OVNIC is a short term addition

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.13-nightly-x86-perfscale-aws-control-plane-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-nightly-x86-perfscale-aws-control-plane-24nodes-periodic periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-perfscale-aws-control-plane-24nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml [hunk 3]
```diff
@@ -86,6 +92,8 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.2xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml:96]: previously we used xlarge, not 2xlarge.

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml]: This is updated resolving the comments.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml [hunk 4]
```diff
@@ -91,10 +101,10 @@ tests:
     env:
       BASE_DOMAIN: perfscale.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "9"
-      COMPUTE_NODE_TYPE: m5.2xlarge
+      COMPUTE_NODE_TYPE: m6i.2xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
       LOKI_USE_SERVICEMONITOR: "false"
-      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
-      SET_ENV_BY_PLATFORM: custom
```
Comment (qiliRedHat) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml:97]: Only when this is set to custom, the overwritten of OPENSHIFT_INFRA_NODE_INSTANCE_TYPE is used. https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh#L639
cc: @liqcui

**Review by qiliRedHat — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml (hunk 4 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86.yaml:97]: +1
Yes, need to specify SET_ENV_BY_PLATFORM: custom.

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes

**Comment by krishvoor** [comment]
From logs, I see this
```
Using control plane instance type: m6i.xlarge
Using compute instance type: m6i.xlarge
Using compute node replicas: 24
Using controlPlane node replicas: 3
```

**Comment by jtaleric** [comment]
```{
        "ciSystem":"PROW",
        "uuid":"5077bb78-28fa-4926-a2c0-b8c6009830e8",
        "releaseStream":"4.16.0-0.nightly",
        "platform":"AWS",
        "clusterType":"self-managed",
        "benchmark":"cluster-density-v2",
        "masterNodesCount":3,
        "workerNodesCount":24,
        "infraNodesCount":3,
        "masterNodesType":"m6i.xlarge",
        "workerNodesType":"m6i.xlarge",
        "infraNodesType":"m6i.xlarge",
        "totalNodesCount":30,
        "clusterName":"ci-op-v55j0d9q-d571b-9g984",
        "ocpVersion":"4.16.0-0.nightly-2024-02-05-130123",
        "networkType":"OVNKubernetes",
        "buildTag":"1754577382020222976",
        "jobStatus":"success",
        "buildUrl":"https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/rehearse-44600-periodic-ci-openshift-qe-XXXXXX-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes/1754577382020222976",
        "upstreamJob":"rehearse-44600-periodic-ci-openshift-qe-XXXXXX-perfscale-ci-main-aws-4.16-nightly-x86-control-plane-24nodes",
        "upstreamJobBuild":"6aef3b6a-3d74-474a-b9b7-3e0df87205db",
        "executionDate":"2024-02-05T19:48:59Z",
        "jobDuration":"2510",
        "startDate":"2024-02-05T19:48:59Z",
        "endDate":"2024-02-05T20:30:49Z",
        "startDateUnixTimestamp":"1707162539",
        "endDateUnixTimestamp":"1707165049",
        "timestamp":"2024-02-05T19:48:59Z",
        "ipsec":"false",
        "fips":"false",
        "encrypted":"false",
        "encryptionType":"",
        "publish":"External",
        "computeArch":"amd64",
        "controlPlaneArch":"amd64"
        }```

**Comment by jtaleric** [comment]
I am good with this change -- my only concern is 4.14 -- if we want to compare to 4.15 switching mid stream might make things messy.

**Comment by krishvoor** [comment]
> I am good with this change -- my only concern is 4.14 -- if we want to compare to 4.15 switching mid stream might make things messy.

We already have "mixed" data giving us m6a/m6i's

**Pull request closed without merging**